geiger/src/graph/NodeWithPort.js

```diff
@@ -242,6 +242,11 @@ export default class NodeWithPort extends Node {
     return ports;
   }
 
+  /**
+   * Validate ports created and then store.
+   * @param {!Array<Port>} ports
+   * @return {!Collection}
```

Comment:
Also what is `Collection`? Are these are included/imported from somewhere else?

Reply:
Since `Collection` is a module already imported via `import ...`, I simply ignore the `@typedef`.